### DISCUSSION TITLE
Subsonic and Transonic 2D NACA 0012 Steady Compressible Simulation

### DISCUSSION THREAD

**Discussion by J-Dario-Arredondo** [su2code/SU2#1713] [Q&A]
I was trying to do a simulation/ analysis for a 2D NACA0012 aerofoil in steady, turbulent, and compressible conditions, using a Spalart-Allmaras turbulence model and 2D RANS equations. 

However, I am struggling to get proper convergence. Therefore, I was wondering if anyone knows what are the best values/ configurations for this simulation are? i.e. CFL number, reference values, numerical methods, and convergence parameters. Since the SU2 tutorials contain only the case of an unsteady compressible NACA0012 and an incompressible steady NACA0012 but not a compressible and steady NACA0012 case.

Thank you very much for your kind support, attention, and time in advanced, I appreciate it.

**Comment by pcarruscag** [comment]
Not sure what grids you are using, but these settings work ok on the TMR grids. It takes quite a few iterations (50k+) on the finest levels because the wake cells are so stretched, so make sure you restart from coarser levels.
[example.zip.txt](https://github.com/su2code/SU2/files/9123214/example.zip.txt)

**Reply by J-Dario-Arredondo** [reply]
Thank you for the support, I think the gridding was the problem, as I was not using the TMR grids. Thank you very much for the support, and will notify if I continue to have more issues regarding this problem.
